code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -70,36 +70,37 @@
 
 	//they have holy water in them? deconversion mode activate! anyone can do it. 'cept cultists O B V I O U S L Y
 	if (!isanycultist(user) && M.reagents?.has_reagent(HOLYWATER) && user.a_intent == I_HELP)
+		user.visible_message("<span class='warning'>[user] [pick(attack_verb)] [M]'s head with \the [src].</span>")
+		playsound(src, "punch", 25, 1, -1)
 		if (M.stat == DEAD)
 			to_chat(user,"<span class='warning'>You cannot deconvert the dead!</span>")
 			return 1
 		if (M.health < 20)
 			to_chat(user,"<span class='warning'>\The [M] is too weak to handle the deconversion ritual, patch them up a bit first.</span>")
 			return 1
+		var/datum/role/cultist/cultist
 		if(iscultist(M))
-			var/datum/role/cultist/cultist = M.mind.GetRole(CULTIST)
+			cultist = M.mind.GetRole(CULTIST)
 			if (cultist.deconversion)
 				to_chat(user,"<span class='warning'>There is already a deconversion attempt undergoing!</span>")
 				return 1
 			else
-				playsound(src, "punch", 25, 1, -1)
+				to_chat(M,"<span class='userdanger'>They are trying to deconvert you!</span>")
+
+		if (do_after(user, M, 5 SECONDS))
+			if(cultist)
 				if (istype(my_rel, /datum/religion/cult))
-					user.visible_message("<span class='warning'>[user] [pick(attack_verb)] [M]'s head with \the [src] and they begin to radiate with light.</span>",
-					"<span class='warning'>You bless [M]'s head with \the [src]. In the name of this [my_rel.deity_name] fanfiction headcanon, Nar-Sie forsake this body and soul!</span>")
+					to_chat(user,"<span class='warning'>In the name of this [my_rel.deity_name] fanfiction headcanon, Nar-Sie forsake this body and soul!</span>")
 				else
-					user.visible_message("<span class='warning'>[user] [pick(attack_verb)] [M]'s head with \the [src] and they begin to radiate with light.</span>",
-					"<span class='warning'>You bless [M]'s head with \the [src]. In the name of [my_rel.deity_name], Nar-Sie forsake this body and soul!</span>")
+					to_chat(user,"<span class='warning'>In the name of [my_rel.deity_name], Nar-Sie forsake this body and soul!</span>")
```

Comment:
This is weird. It sounds like someone's spoken monolog but it's a system message to the user. It made more sense with "You bless [target]'s head." in front.